README.md

```diff
@@ -38,6 +38,7 @@ dockerCompose.isRequiredBy(test) // hooks 'dependsOn composeUp' and 'finalizedBy
 
 dockerCompose {
     // useComposeFiles = ['docker-compose.yml', 'docker-compose.prod.yml'] // like 'docker-compose -f <file>'
+    // captureContainersOutput = true // prinnts output of all containers to Gradle output - very useful for debugging
```

Comment:
I know I am late to the party ;) but you can change this in master:
I see a typo and I would actually prefer `captureContainerOutput` name of the parameter.